controller/workitem_blackbox_test.go

```diff
@@ -559,6 +559,37 @@ func (s *WorkItemSuite) TestUpdateWorkitemWithoutReorder() {
 	assert.Equal(s.T(), wi.Data.Attributes[workitem.SystemOrder], updated.Data.Attributes[workitem.SystemOrder])
 }
 
+// TestReorderDoesNotUpdateAnyOtherWIFields tests that no other workitem fields are updated while reorder
+func (s *WorkItemSuite) TestReorderDoesNotUpdateOtherWIFields() {
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment())
+
+	// create workitems
+	payload := minimumRequiredCreateWithTypeAndSpace(fxt.WorkItemTypes[0].ID, fxt.Spaces[0].ID)
+	originalTitle := "Original Title"
+	payload.Data.Attributes[workitem.SystemTitle] = originalTitle
+	payload.Data.Attributes[workitem.SystemState] = workitem.SystemStateClosed
+	_, workitem1 := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *payload.Data.Relationships.Space.Data.ID, &payload)
```

Comment:
You can use test fixture to create the initial work item. That would remove all the code from line 566 to 576

Reply:
@jarifibrahim The test fixture gives me a workitem of `workitem.Workitem` type but I need `app.WorkItem` to create reorder payload.